examples/src/chains/chat_vector_db.ts

```diff
@@ -1,9 +1,51 @@
 import { OpenAI } from "langchain/llms";
-import { ChatVectorDBQAChain } from "langchain/chains";
+import {
+  ChatVectorDBQAChain,
+  StuffDocumentsChain,
+  LLMChain,
+} from "langchain/chains";
 import { HNSWLib } from "langchain/vectorstores";
 import { OpenAIEmbeddings } from "langchain/embeddings";
 import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
 import * as fs from "fs";
+import { PromptTemplate } from "langchain";
+import { KeyValueOutputParser } from "langchain/output_parsers";
+
+const template = `Given the following extracted parts of a long document and a question, create a final answer with references ("SOURCES"). 
```

Comment:
This is temporary just for me to test it, we should move the prompt into the main package